### PR TITLE
Add browser table release changelog entry

### PR DESCRIPTION
**Pull request by rileyhgrant**
Automatically generated by Decap CMS

---

Edit: links to preview for convenience
- https://gnomad.broadinstitute.org/news/preview/166/2024-08-release-gnomad-browser-tables/
- https://gnomad.broadinstitute.org/news/preview/166/changelog/

### CODE REVIEW

**Comment by rileyhgrant** [comment]
Related: #165

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The Hail Tables that underlie the gnomAD browser are now available for public use. These tables closely mirror the structure of the browser, enabling users familiar with the gene and variant pages to easily access the full set of underlying data in a format consistent with the browser's organization.
+
+For more information about these tables, please refer to our [help text](http://gnomad.broadinstitute.org/help/v4-browser-hts) on the browser release tables. 
+
+To download these tables, visit our [downloads page](http://gnomad.broadinstitute.org/downloads#v4-browser-tables).
+
+<!-- end_excerpt -->
+
+**Three tables are now available**
```
Comment (ch-kr) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:15]:
```suggestion
**Three tables are now available:**
```
(optional)

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- v4.1 browser variants Hail Table
+- GRCh38 browser gene models Hail Table
+- GRCh37 browser gene models Hail Table
+
+### gnomAD v4.1 Exome/Genome/Joint Variant Hail Table
```
Comment (ch-kr) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:21]:
```suggestion
### gnomAD v4.1 browser variants Hail Table
```
for consistency with the bullet point above

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The browser variant table consolidates data from exomes, genomes, and joint frequencies into a single, unified table. Frequency information is stored as a dictionary, the key-value pairs allow for easy access to specific frequencies.
```
Comment (ch-kr) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:23]:
```suggestion
The browser variant table consolidates data from the gnomAD [exome](https://gnomad.broadinstitute.org/downloads#v4-variants), [genome](https://gnomad.broadinstitute.org/downloads#v4-variants), and [joint frequency](https://gnomad.broadinstitute.org/downloads#v4-joint-freq-stats) releases into a single, unified table. Frequency information is stored as a dictionary, and the key-value pairs allow for easy access to specific frequencies.
```

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+### gnomAD GRCh38/GRCh37 Gene Model Hail Tables
```
Comment (ch-kr) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:25]:
```suggestion
### GRCh38/GRCh37 Gene Models Hail Tables
```

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+For more information about these tables, please refer to our [help text](http://gnomad.broadinstitute.org/help/v4-browser-hts) on the browser release tables. 
+
+To download these tables, visit our [downloads page](http://gnomad.broadinstitute.org/downloads#v4-browser-tables).
+
+<!-- end_excerpt -->
+
+**Three tables are now available**
+
+- v4.1 browser variants Hail Table
```
Comment (ch-kr) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:17]:
```suggestion
- gnomAD v4.1 browser variants Hail Table
```

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- GRCh38 browser gene models Hail Table
+- GRCh37 browser gene models Hail Table
+
+### gnomAD v4.1 Exome/Genome/Joint Variant Hail Table
+
+The browser variant table consolidates data from exomes, genomes, and joint frequencies into a single, unified table. Frequency information is stored as a dictionary, the key-value pairs allow for easy access to specific frequencies.
+
+### gnomAD GRCh38/GRCh37 Gene Model Hail Tables
+
+The gene models tables aggregate relevant information from multiple sources into a single table. For each gene, data is included from GENCODE, HGNC, MANE transcripts, GTEx, and various gnomAD secondary analyses.
```
Comment (ch-kr) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:27]:
```suggestion
The gene models tables aggregate relevant information from multiple sources into a single table. For each gene, data is included from GENCODE, HGNC, and MANE transcripts. Information from GTEx, and various gnomAD secondary analyses are available in the GRCh37 table but are not yet available in the GRCh38 table.
```

**Review by ch-kr — COMMENTED** [review]
thank you for the links -- super handy! just few minor comments to increase clarity

File: content/changelog/2024-08-release-gnomad-browser-tables.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+title: Release gnomAD browser tables
+date: 2024-08-20
```
Comment (mattsolo1) [inline content/changelog/2024-08-release-gnomad-browser-tables.md:3]: Update date

**Review by mattsolo1 — APPROVED** [review]

**Pull request merged**